doc/ref_model/chapters/chapter04.md

```diff
@@ -328,8 +328,8 @@ Table 4-14: Reserved
 | e.vim.pm.001 | Time to create Virtual Compute for a given VNF | Max ms |  |
 | e.vim.pm.002 | Time to delete Virtual Compute of a given VNF | Max ms |  |
 | e.vim.pm.003 | Time to start Virtual Compute of a given VNF | Max ms |  |
-| e.vim.pm.004 | Time to stop Virtual Compute of a given VNF | Max ms |  |
-| e.vim.pm.005 | Time to pause Virtual Compute of a given VNF | Max ms |  |
+| e.vim.pm.004 | Time to stop Virtual Compute of a given VNF | Max ms | <sup>1)</sup> |
+| e.vim.pm.005 | Time to pause Virtual Compute of a given VNF | Max ms | <sup>2)</sup> |
```

Comment:
Can I suggest we change "Virtual Compute" to allow for bare metal scenarios. I'm not 100% what the intent of these metrics are, but perhaps "execution environment" (and we have something in the glossary that defines that as the physical or virtual infrastructure, plus an operating system)?

Reply:
@tomkivlin As you know, kubernetes also has ephemeral storage albeit not of the type in 
| e.nfvi.cap.003 | Total per-instance (ephemeral) storage | GB | Max storage in GB that can be assigned to a single VNFC by NFVI | In Kubernetes there are two types of non-persistent storage - that which is used to execute the containers, but also Kubernetes Volumes that are not Persistent Volumes, how should we deal with these? |

Reply:
@tomkivlin "Virtual Compute resource" from the glossary covers both the bare metal container and the VM case. I've modified e.vim.pm.001-005 to use "Virtual Compute resource".

Reply:
@tomkivlin can you check if we can resolve these?